dapr/proto/runtime/v1/dapr.proto

```diff
@@ -193,6 +193,15 @@ service Dapr {
   rpc RaiseEventWorkflowBeta1 (RaiseEventWorkflowRequest) returns (google.protobuf.Empty) {}
   // Shutdown the sidecar
   rpc Shutdown (ShutdownRequest) returns (google.protobuf.Empty) {}
+
+  // Create and schedule a job
+  rpc ScheduleJobAlpha1(ScheduleJobRequest) returns (google.protobuf.Empty) {}
```

Comment:
1. Dont' return Emtpy. Instead, create a new message `ScheduleJobAlpha1Response` and make it empty. Same below.
2. Don't forget `Alpha` in the Request type

Reply:
I don't see this being done for other protos, I'd prefer consistency. For example:

```
  rpc SubtleGetKeyAlpha1(SubtleGetKeyRequest) returns (SubtleGetKeyResponse);
  // SubtleEncryptAlpha1 encrypts a small message using a key stored in the vault.
  rpc SubtleEncryptAlpha1(SubtleEncryptRequest) returns (SubtleEncryptResponse);
```

Reply:
Ok. At least don't use `google.protobuf.Empty` please! This also helps with the tracing code as it expects a type for each message.

Reply:
Agree on not using Empty

Reply:
Once a type is defined, it then needs to be added here for tracing: https://github.com/dapr/dapr/blob/master/pkg/proto/runtime/v1/dapr_tracing.go (it's ok to leave implementation as TODO for now)

Reply:
I have the TODOs in the `dapr_tracing` file in this PR currently. I can update it to not use Empty.

Reply:
Yes do not use Empty, because it makes things easier in the future. Ok to leave TODOs in the tracing file for now

Reply:
https://github.com/cicoyle/dapr/pull/71